src/tensorrt_lightnet_node.cpp

```diff
@@ -21,17 +21,11 @@ namespace tensorrt_lightnet
 
         trt_lightnet_ = std::make_unique<tensorrt_lightnet::TrtLightNet>(model_cfg, model_weights);
 
-        timer_ = rclcpp::create_timer(this, get_clock(), 100ms, std::bind(&TrtLightNetNode::onConnect, this));
-
-        image_pub_ = image_transport::create_publisher(this, "~/out/image");
-    }
-
-    void TrtLightNetNode::onConnect()
```

Comment:
@manato I guess @chickenta2ta wanted to implement `onConnect()` like [this](https://github.com/autowarefoundation/autoware.universe/blob/5fed746365fd647c61eae400fbddc304088b9d4c/perception/tensorrt_yolox/src/tensorrt_yolox_node.cpp#L118-L131)

Reply:
@wep21 
Thank you very much for your prompt comment! 
Yes, I agree with your guess (actually, I also came up with that implementation). In my understanding, creating subscribers in such a timer function aims to reduce the system load if no nodes subscribe to the results. But, as a side effect, the node will not start working as long as result-subscribers do not exist (and sometimes it makes users confused, I think). For this reason, I omitted the timer and made this node start working just after node execution for simplicity.

Since I don't have a strong intention other than the above, I think making `onConnect()` similar to the other node should also be fine!

Reply:
Thank you both for the comments and fixes!
As @wep21 mentioned above, my intention of using a timer function was to mimic the implementation of other nodes.
I understand not using a timer function results in simpler behavior, so I'm cool with this implementation without a timer function as @manato suggested.